tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>I would rename the section to Momentum balance and skip the sentence "In the case of the mechanics problem we utilize the momentum balance model class. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>